plugins/destination/mysql/client/read.go

```diff
@@ -0,0 +1,95 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func (*Client) createResultsArray(table *schema.Table) []any {
+	results := make([]any, 0, len(table.Columns))
+	for _, col := range table.Columns {
+		switch col.Type {
+		case schema.TypeBool:
+			var r *bool
+			results = append(results, &r)
```

Comment:
```suggestion
			results = append(results, new(*bool))
```

Reply:
What's the added value of doing it this way? It won't be consistent with the rest of the code